x-pack/plugins/threat_intelligence/common/test/utils.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Use this to query elements by test-subj
+ * @param testSubject test subject to query elements by
+ * @returns
+ */
+export const getByTestSubj = (testSubject: string): HTMLElement =>
```

Comment:
if the method below is unused, do we really need this one?

Reply:
Removed